lua/wire/server/debuggerlib.lua

```diff
@@ -1,4 +1,8 @@
-local formatPort = {}
+local formatPort = setmetatable({}, { __index = function(t, k)
+	return rawget(t, k) or function() return k end
```

Comment:
__index will only call if the table doesn't contain index 'k' so the rawget will just always return nil